content/applications/general/iot.rst

```diff
@@ -1,14 +1,75 @@
-:nosearch:
 :show-content:
 :hide-page-toc:
-:show-toc:
+:nosearch:
 
 ========================
 Internet of Things (IoT)
 ========================
 
+Odoo IoT allows to connect physical devices such as barcode scanners, receipt printers, payment
+terminals, measurement tools, etc. to an Odoo database using an IoT system.
+
+The following IoT systems are supported:
+
+- :doc:`IoT box <iot/iot_box>`: micro-computer, plug-and-play device (IoT program pre-installed);
+- :doc:`Windows virtual IoT <iot/windows_iot>`: Windows program that can be installed on a Windows
+  computer.
+
+.. important::
+   An IoT box subscription is required to use IoT systems with a secured connection.
```

Comment:
I'm not that sure of the phrasing here. It is true but in practice it would suggest that it is not mandatory if they don't care about secured connection. While it is true if you have the knowledge on how to handle this. I don't think trying to justify this "feature" with this "reason" would be wise here.
Maybe more something like this:
"It is required to use an IoT box subscription for production usage of the IoT box" ..seealso applications/general/iot/iot_advanced/https_certificate_iot.html#https-certificate-generation